examples/whisper/user_script.py

```diff
@@ -5,8 +5,147 @@
 from pathlib import Path
 
 import numpy as np
-from onnxruntime.transformers.models.whisper.whisper_decoder import WhisperDecoderInputs
-from onnxruntime.transformers.models.whisper.whisper_encoder_decoder_init import WhisperEncoderDecoderInitInputs
+from onnxruntime.transformers.models.t5.past_helper import PastKeyValuesHelper
+from onnxruntime.transformers.models.whisper.whisper_decoder import WhisperDecoder, WhisperDecoderInputs
+from onnxruntime.transformers.models.whisper.whisper_encoder_decoder_init import (
```

Comment:
Could we just copy WhisperDecoder etc., here in the example folder ?

Reply:
I thought that would be duplicated code from onnxruntime. It will be too much to copy all codes from onnxruntime repo.

Reply:
Neither onnxruntime nor Olive are supposed to be the vendor of the WhisperDecoder etc., components. If we copy this here in the example folder then it becomes clear to the user that this is how they can provide their own custom components. 

If we can establish this then eventually we can remove the WhisperDecoder etc., from the onnxruntime repository.

Reply:
Added custom components implement.